tb.go

```diff
@@ -0,0 +1,35 @@
+package merkletree
+
+import "github.com/coniks-sys/libmerkleprefixtree-go/crypto"
+
+type TemporaryBinding struct {
+	index []byte
+	value []byte
+	sig   []byte
+}
+
+func (pad *PAD) NewTemporaryBinding(key string, value []byte) *TemporaryBinding {
```

Comment:
Can we move this to pad.go though, to be closer to `PAD`'s other methods?  And maybe rename it to just `.TB()`.